### PR TITLE
Add link about flycheck-prospector extension

### PR DESCRIPTION
**Pull request by chocoelho**
Adds flycheck-prospector on Python supported extensions.

### CODE REVIEW

File: doc/community/extensions.rst [hunk 1]
```diff
@@ -133,6 +133,7 @@ Python
 
 * :gh:`Wilfred/flycheck-pyflakes` adds a Python syntax checker using Pyflakes.
 * :gh:`msherry/flycheck-pycheckers` adds a checker for Python that can run multiple syntax checkers simultaneously (Pyflakes, PEP8, mypy2/3, etc.).
+* :gh:`chocoelho/flycheck-prospector` adds prospector checker for Python syntax.
```
Comment (fmdkdd) [inline doc/community/extensions.rst]: Could you add a link to `prospector`?  While you are here, you could fix the missing link for `Pyflakes` above (I think adding the underscore `Pyflakes_` should do it).

Comment (chocoelho) [reply doc/community/extensions.rst]: np, just did it. I also took the liberty to do the same for Mypy, if that's not an issue...

**Comment by fmdkdd** [comment]
Thanks!  That's great.  Please just squash your commits and I will merge.

**Comment by chocoelho** [comment]
Done

**Pull request merged**